src/android/Print.java

```diff
@@ -259,6 +259,12 @@ private boolean createReceiptData(Order order) {
 			if (order.customer_address != null && !order.customer_address.isEmpty())
 				builder.append("Address: " + order.customer_address + "\n");
 
+			if (order.handoff_provider != null && !order.handoff_provider.isEmpty())
+            	builder.append("Marketplace Provider: " + order.handoff_provider + "\n");
+
+			if (order.delivery_id != 0 && (order.delivery_type != null && order.delivery_type.equals("doordash")))
```

Comment:
I'd prefer to see this as provider-independent code, rather than being localized to DD, since we know we're adding other providers (Uber, Lyft, Skip, etc.)

Could have a field like delivery_provider_display with the display-formatted value